File: trail2/함수를 이용한 소수 판별/decimal-decisions-using-functions.cpp
#include <iostream>
using namespace std;

bool isPrime(int num) {
    bool is_prime = true;

    for (int i = 2; i < num; i++) {
        if (num % i == 0) {
            is_prime = false;
            break;
        }
    }

    return is_prime;
}

int main() {
    // Please write your code here.
    int a, b;
    cin >> a >> b;

    int sum = 0;
    for (int i = a; i <= b; i++) {
        if (isPrime(i)) 
            sum += i;
    }

    cout << sum << endl;

    return 0;
}

File: trail2/함수를 이용한 소수 판별/decimal-decisions-using-functions.py
a, b = tuple(map(int, input().split()))

def is_prime(num):
    for i in range(2, num):
        if num % i == 0:
            return False
    return True

sum = 0
for i in range(a, b+1):
    if (is_prime(i)):
        sum += i

print(sum)